src/Common/RJSF/templates/ButtonTemplates/AddButton.tsx

```diff
@@ -32,23 +31,23 @@ export const AddButton = ({
 
     return (
         <div className="flexbox flex-justify-start">
-            <Tippy
+            <Tooltip
+                showOnTruncate
```

Comment:
should be true by default

Reply:
Please see https://github.com/devtron-labs/devtron-fe-common-lib/pull/235#discussion_r1709990752